test/core/utils/preload-media.js

```diff
@@ -0,0 +1,76 @@
+/* global Promise */
+
+describe('axe.utils.preloadMedia', function() {
```

Comment:
You're not testing what happens when something errors out.

Reply:
Removed the only reject that was here - More info - https://github.com/dequelabs/axe-core/pull/1958#discussion_r364315256.